src/test/scala/nl.knaw.dans.easy.solr4files/UpdateServletSpec.scala

```diff
@@ -80,7 +80,7 @@ class UpdateServletSpec extends TestSupportFixture
 
   it should "return NOT FOUND if uuid is missing" in {
     post("/update/pdbs/") {
-      body should startWith("""Requesting "POST /update/pdbs/" on servlet "" but only have:""")
+      body should startWith("""Requesting "POST \/update\/pdbs\/" on servlet "" but only have:""")
```

Comment:
Is this escape an "improvement :-(" by scala-v12?

Reply:
It has probably to do with Scalatra, rather than Scala 2.12. I still need to look into that. Thanks for the reminder!